### PR TITLE
ECCW-572: Initial dev setup

### PR DESCRIPTION
**Pull request by siliconmeadow**
DDEV setup, README, composer setup. Essex theme in place.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+## Quick start for local development
+
+Use [DDEV](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/).
+
+1. Clone this repo.
+2. Run `ddev start` in the project root.
+3. Run `ddev composer install` to install the project dependencies.
+4. Run `ddev drush si localgov_microsites`.
+
+The controller site is then available at https://ecc-microsites.ddev.site
```
Comment (MatthewWilkes) [inline README.md:14]: Following these instructions gives a microsites site, but without the Essex theme

**Review by MatthewWilkes — DISMISSED** [review]
Could you also add the gitlab-ci file to build and push these images?

File: README.md (hunk 1 above)
Comment (siliconmeadow) [reply README.md:14]: @MatthewWilkes - The `localgov_microsites` profile is unaware of the essex theme, and also I noticed the theme is mentioned in another ticket (ECCW-570). What I'll do is have composer swap `web/profiles/contrib/localgov_microsites/localgov_microsites.info.yml` for a modified version in that upcoming PR, if that's OK with you.

**Comment by siliconmeadow** [comment]
> Could you also add the gitlab-ci file to build and push these images?

@MatthewWilkes - It will be in the next ticket/PR which is in progress. ECCW-574.

**Review by MatthewWilkes — APPROVED** [review]

**Pull request merged**